<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>measuring_tool</class>
 <widget class="QDialog" name="measuring_tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>382</width>
    <height>404</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Measurement Tool</string>
  </property>
  <widget class="QLabel" name="info_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDistances">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAngles">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Angles:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableAngles">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablePositions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPositions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Atom Positions</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDistances">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Distances:</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_text_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
